kfdef/kfctl_openshift.yaml

```diff
@@ -110,6 +110,11 @@ spec:
         name: manifests
         path: odh-dashboard
     name: odh-dashboard
+  - kustomizeConfig:
```

Comment:
I'd remove this from the sample kfctl file

Reply:
Yeah, it's not really used anywhere in red-hat-data-services and might be an unnecessary diff from opendatahub-io